setup.py

```diff
@@ -23,6 +23,7 @@
     "requests>=2.12.4",
     "rlp>=0.4.7",
     "toolz>=0.8.2",
+    "ethereum-tester==0.1.0-beta.1",
```

Comment:
Thoughts on this?  It doesn't add any system dependencies and is pure python.  The `web3.providers.eth_tester.EthereumTesterProvider` requires it.